git-tool-config.ps1

```diff
@@ -40,3 +41,11 @@ if ($defaultServiceLine -ne '') {
 } else {
     Write-Host "Using previous default service line: $($oldConfig.defaultServiceLine)"
 }
+
+if (-not $disableAtomicPush) {
```

Comment:
Omission of this flag enables atomic push - which may not be what is desired, given the documentation. The rest of the switches choose to enable or disable.

I'd also rather the "flag" be a flag rather than the string - getting the parameter could be part of the "get config" function, but this allows for abuse inside the config itself.

Reply:
Changes pushed to address this and not assume to enable atomic push even though by default it is enabled if the config value is unset.